packages/qscintilla/build.sh

```diff
@@ -0,0 +1,41 @@
+TERMUX_PKG_HOMEPAGE=https://riverbankcomputing.com/software/qscintilla
+TERMUX_PKG_DESCRIPTION="QScintilla is a port to Qt of the Scintilla editing component"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="Simeon Huang <symeon@librehat.com>"
+TERMUX_PKG_VERSION=2.12.1
+TERMUX_PKG_REVISION=1
+TERMUX_PKG_SRCURL="https://www.riverbankcomputing.com/static/Downloads/QScintilla/${TERMUX_PKG_VERSION}/QScintilla_src-${TERMUX_PKG_VERSION}.tar.gz"
+TERMUX_PKG_SHA256=a7331c44b5d7320cbf58cb2382c38857e9e9f4fa52c405bd7776c8b6649836c2
+TERMUX_PKG_DEPENDS="qt5-qtbase"
+# qttools is only needed to build Qt Designer's plugins
+TERMUX_PKG_BUILD_DEPENDS="qt5-qtbase-cross-tools, qt5-qttools"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_NO_STATICSPLIT=true
+
+termux_step_configure () {
+    for i in src designer; do
+        cd "${TERMUX_PKG_SRCDIR}/${i}" && {
+            "${TERMUX_PREFIX}/opt/qt/cross/bin/qmake" \
+                -spec "${TERMUX_PREFIX}/lib/qt/mkspecs/termux-cross"
+        }
+    done
+    unset i
+}
+
+termux_step_make() {
+    cd "${TERMUX_PKG_SRCDIR}/src" && {
+        make -j "${TERMUX_MAKE_PROCESSES}"
+    }
+}
```

Comment:
Why does termux_step_make need to be overridden?

Reply:
It needs to be done in the extra `/src` subdirectory

Reply:
Sorry, missed that. You can set `TERMUX_PKG_EXTRA_MAKE_ARGS=" -C src` instead, as done by some packages (for example [[1]](https://github.com/termux/termux-packages/blob/3622ee1fe6cc2b1f52a9acd2f83b7dba2e345247/packages/wireguard-tools/build.sh#L9), [[2]](https://github.com/termux/termux-packages/blob/master/packages/liblmdb/build.sh#8), [[3]](https://github.com/termux/termux-packages/blob/3622ee1fe6cc2b1f52a9acd2f83b7dba2e345247/packages/kakoune/build.sh#L10)). IMO that is preferable to overriding the entire function